alkira/resource_alkira_service_f5_lb.go

```diff
@@ -189,25 +189,26 @@ func resourceAlkiraF5LoadBalancer() *schema.Resource {
 							Type:        schema.TypeString,
 							Required:    true,
 						},
-						"deployment_option": {
-							Description: "The deployment option of the F5 LB instance," +
-								" can be one of `ONE_BOOT_LOCATION` or `TWO_BOOT_LOCATION`." +
-								" Only required when license_type is `BRING_YOUR_OWN`.",
-							ValidateFunc: validation.StringInSlice(
-								[]string{"ONE_BOOT_LOCATION", "TWO_BOOT_LOCATION"},
-								false),
-							Type:     schema.TypeString,
-							Optional: true,
-						},
+						// "deployment_option": {
```

Comment:
Just remove this for now. When we really need to do this, then bring it back as a new optional field.